sdk/storage/Azure.Storage/README.md

```diff
@@ -0,0 +1,73 @@
+# Azure Storage meta-package client library for .NET
+
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][docs] | [REST API documentation][rest_docs] | [Product documentation][product_docs]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Storage meta-package for .NET with [NuGet][nuget]:
+
+```Powershell
+dotnet add package Azure.Storage
+```
+
+### Prerequisites
```

Comment:
Should we bother with all these empty or copy/pasted sections and just link folks to the other readmes?